roq-deployment/ansible/gimme-roq/files/pomAlt.xml

```diff
@@ -0,0 +1,311 @@
+	        </os>
+            </activation>
+            <properties>
+	        <native.os>Windows</native.os>
+	        <native.path>lib/jzmq.dll</native.path>
+	        <!-- Use platform-specific path separators here: -->
+	        <native.library-path>lib</native.library-path>
+            </properties>
+        </profile>
+        <profile>
+            <id>Mac</id>
+            <activation>
+	        <os>
+	            <family>mac</family>
+	        </os>
+            </activation>
+            <properties>
+	        <native.os>${os.name}</native.os>
+	        <native.path>src/.libs/libjzmq.dylib</native.path>
+	        <!-- Use platform-specific path separators here: -->
+	        <native.library-path>src/.libs/</native.library-path>
+            </properties>
+        </profile>
+        <profile>
+            <id>os-distro</id>
+            <activation>
+	        <property>
+	            <name>os.distro</name>
+	        </property>
+            </activation>
+            <properties>
+                <native.distro>-${os.distro}</native.distro>
+            </properties>
+        </profile>
+        <profile>
+            <id>zmq-version</id>
+            <activation>
+	        <property>
+	            <name>zmq.version</name>
+	        </property>
+            </activation>
+            <properties>
+                <native.zmq.version>-${zmq.version}</native.zmq.version>
+            </properties>
+        </profile>
+        <profile>
+            <id>release</id>
+            <properties>
+	        <native.path.resolved>..${file.separator}..${file.separator}${native.path}</native.path.resolved>
+	        <native.library-path.resolved>..${file.separator}..${file.separator}${native.library-path}</native.library-path.resolved>
+            </properties>
+        </profile>
+        <profile>
+            <id>deploy-local-maven</id>
+            <distributionManagement>
+                <repository>
+                    <id>maven.releases</id>
+                    <url>${env.RELEASE_REPO}</url>
+                </repository>
+                <snapshotRepository>
+                    <id>maven.snapshots</id>
+                    <url>${env.SNAPSHOT_REPO}</url>
+                </snapshotRepository>
+            </distributionManagement>
+        </profile>
+    </profiles>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit.version}</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <sourceDirectory>src</sourceDirectory>
+        <testSourceDirectory>test/src</testSourceDirectory>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
+<configuration> <skip>true</skip> </configuration>
+                <configuration> <skip>true</skip> </configuration>
```

Comment:
Why is this file here? There are two similar pom files in this pull request, so one of them is probably redundant. I suppose it is this one. Lines 165 and 166 are the same.